packages/search-ui/src/URLManager.js

```diff
@@ -78,8 +83,13 @@ function stateToParams({
   if (sortField) {
     params["sort-field"] = sortField;
     params["sort-direction"] = sortDirection;
+    delete params["sort"];
+  }
+  if (sortList && sortList.length > 0) {
+    delete params["sort-field"];
+    delete params["sort-direction"];
+    params["sort"] = sortList;
   }
```

Comment:
@JasonStoltz thoughts on this?

Reply:
Just looking at this code in isolation. See my comments inline below.

```js
  const params = {};
  // params === {}
  if (current > 1) params.current = current;
  // params === { current: 1 }
  if (searchTerm) params.q = searchTerm;
  // params === { current: 1, q: 'text' }
  if (resultsPerPage) params.size = resultsPerPage;
  // params === { current: 1, q: 'text', size: 10 }
  if (filters && filters.length > 0) {
    params["filters"] = filters;
    // params === { current: 1, q: 'text', size: 10, filters: [...] }
  }
  if (sortField) {
    params["sort-field"] = sortField;
    params["sort-direction"] = sortDirection;
    // params === { current: 1, q: 'text', size: 10, filters: [...] }
    // params contains no 'sort' property here, so deleting it has no effect.
    delete params["sort"];
  }
  if (sortList && sortList.length > 0) {
    // params === { current: 1, q: 'text', size: 10, filters: [...], sort-field: 'title', sort-direction: 'asc' }
    // We *just* added sort-field and sort-direction. Instead of deleting them here, we could just check
    // to see if 'sortList' is present before we add them, and never add them to begin with
    delete params["sort-field"];
    delete params["sort-direction"];
    params["sort"] = sortList;
  }
```

My proposal I *think* is functionally equivalent, I'm just checking for certain things before adding them, rather than adding them and then deleting them.

There's unit tests for this, I'm fairly certain they'll still pass with this change.

It's not critical, it's just a little easier on the eyes.

Reply:
Sorry, I am not sure what I was thinking, you are right.